test/xpu_api/utilities/meta/meta.trans/meta.trans.other/underlying_type.pass.cpp

```diff
@@ -127,18 +130,21 @@ kernel_test2(sycl::queue& deviceQueue)
     });
 #endif
 }
+#endif
 
 int
 main()
 {
     sycl::queue deviceQueue = TestUtils::get_test_queue();
     kernel_test1(deviceQueue);
 
+#if !_PSTL_ICC_TEST_UNDERLYING_TYPE_BROKEN
     const auto device = deviceQueue.get_device();
     if (TestUtils::has_type_support<double>(device))
     {
         kernel_test2(deviceQueue);
     }
+#endif
```

Comment:
I propose to wrap the entire code block with `TEST_STD_VER >= 20 && !_PSTL_ICC_TEST_UNDERLYING_TYPE_BROKEN`

We can skip making copy of the device as well in this case.

Reply:
Fixed